### PR TITLE
Update etl modules for v7, #8567

### PR DESCRIPTION
**Pull request by njkim**
### Types of changes

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Issues Solved

#8567 

### Checklist

-   [ ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @

### CODE REVIEW

File: arches/app/etl_modules/branch_csv_importer.py [hunk 1]
```diff
@@ -116,6 +117,11 @@ def create_tile_value(self, cell_values, data_node_lookup, node_lookup, row_deta
                     config["nodeid"] = nodeid
                 except TypeError:
                     config = {}
+                if datatype == "string":
+                    try:
+                        source_value = ast.literal_eval(source_value)
```
Comment (chiatt) [inline arches/app/etl_modules/branch_csv_importer.py:122]: This check actually belongs in the string datatype's transform_value_for_tile method. If it's moved into transform_value_for_tile, then this same block can be removed from `import_single_csv` as well.

**Review by chiatt — CHANGES_REQUESTED** [review]
Overall this looks good. There is just some datatype specific logic that should be moved into the string datatype class.

File: arches/app/etl_modules/branch_csv_importer.py (hunk 1 above)
Comment (njkim) [reply arches/app/etl_modules/branch_csv_importer.py]: Moved, thanks.

**Review by chiatt — APPROVED** [review]
Looks good to me!

**Pull request merged**